internal/clients/interface.go

```diff
@@ -143,5 +143,5 @@ type GCP interface {
 	ListAllRegions(ctx context.Context) ([]Region, error)
 
 	// InsertInstances launches one or more instances
```

Comment:
would be nice to document the return parameters as there is more now, it might be handy to know what those are.